chat/v1/streaming.proto

```diff
@@ -147,6 +148,12 @@ message Event {
     uint64 guild_id = 2;
     uint64 channel_id = 3;
   }
+  message PermissionUpdated {
```

Comment:
missing a way to subscribe to this

Reply:
I was thinking that using `SubscribeToGuild` would mean that you are also subscribed to this. Servers would just calculate which users need to be sent a `PermissionUpdated` event when someone uses `set_permission`.

Reply:
document that then